org.eclipse.lsp4e.test/src/org/eclipse/lsp4e/test/semanticTokens/SemanticTokensTestUtil.java

```diff
@@ -36,9 +36,9 @@ public class SemanticTokensTestUtil {
 
 	public static List<Integer> keywordSemanticTokens() {
 		List<List<Integer>> expectedTokens = new ArrayList<>();
-		expectedTokens.add(Arrays.asList(0,0,4,1,0));
-		expectedTokens.add(Arrays.asList(3,0,4,1,0));
-		expectedTokens.add(Arrays.asList(0,9,7,1,0));
+		expectedTokens.add(Arrays.asList(0,0,4,0,0));
```

Comment:
You need to increase as well the version number of the test plugin for the commit to be buildable.